experimental/dds/tree2/api-report/tree2.api.md

```diff
@@ -221,7 +221,7 @@ export interface BindTree<T = BindTreeDefault> extends PathStep {
 // @alpha
 export type BindTreeDefault = BindTree;
 
-// @alpha
+// @beta
 export type Brand<ValueType, Name extends string> = ValueType & BrandedType<ValueType, Name>;
```

Comment:
This is exported, meaning its not inside our internal types scope. It should be moved to avoid us stabilizing it unnecessarily.

Reply:
Done, but that breaks doc links (and there is no way to fix them) so I remove them.